app/src/main/java/com/summer/internship/tvtracker/Login/LoginActivity.kt

```diff
@@ -3,19 +3,28 @@ package com.summer.internship.tvtracker.Login
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.text.TextUtils
+import android.util.Log
 import android.util.Patterns
 import android.widget.Toast
+import androidx.core.os.bundleOf
+import androidx.navigation.Navigation.findNavController
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseAuthMultiFactorException
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
 import com.summer.internship.tvtracker.MainActivity
+import com.summer.internship.tvtracker.R
 import com.summer.internship.tvtracker.Register.RegisterActivity
 import com.summer.internship.tvtracker.databinding.ActivityLoginBinding
 
 class LoginActivity : AppCompatActivity() {
```

Comment:
We also need to keep the user logged. In case login/register was succesful, closing the app and re-opening it should keep the user. The Splash Screen should navigate to MainActivity in this case